src/components/core/ApplicationNavigationBar.tsx

```diff
@@ -1,25 +1,121 @@
-import { Navbar, Text } from "@mantine/core";
+import { Navbar, List } from "@mantine/core";
+import Link from "next/link";
+import React, { useState } from "react";
+import { FaChevronDown } from "react-icons/fa";
+
+interface NavigationItem {
```

Comment:
Yeah, this one is already defined in the `Application Header` as well, let's not duplicate here

https://github.com/stamford-syntax-club/stamfordcenter-frontend/blob/main/src/components/core/ApplicationHeader.tsx#L14C3-L14C3

Reply:
yes sirr